### PR TITLE
Install instructions

### PR DESCRIPTION
**Pull request by bedroesb**
The purpose of this PR is to add documentation in how to install the necessary tools and workflows on a galaxy instance.

### CODE REVIEW

File: install/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Install tools and workflows on a Galaxy instance
+
+
```
Comment (bgruening) [inline install/README.md]: can you mention that you need ephemeris please ... pip install ...

Comment (bgruening) [reply install/README.md]: And a short introduction ... given ephemeris and the following files you can install all needed tools into any given Galaxy instance which you have admin access to ...

Comment (bedroesb) [reply install/README.md]: I'm on it ;)

**Comment by dannon** [comment]
@bedroesb I pushed a commit to fix the link seen at https://deploy-preview-52--gifted-liskov-112081.netlify.com/deploy/, but it looks like (I'm guessing) there's some sort of collision here.  I'll figure it out though.

**Comment by bedroesb** [comment]
@dannon Thank you! I know that the deploy link was not working, but couldn't immediately figure out why.

**Comment by dannon** [comment]
@bedroesb Good to go now.

**Comment by bgruening** [comment]
Cool, thanks a lot @bedroesb. Please subscribe to this repo, we need to make sure to mirror the latest versions of the workflow also to this repo.

**Pull request merged**